lib/ably/modules/conversions.rb

```diff
@@ -115,5 +115,38 @@ def ensure_supported_payload(payload)
 
       raise Ably::Exceptions::UnsupportedDataType.new('Invalid data payload', 400, Ably::Exceptions::Codes::INVALID_MESSAGE_DATA_OR_ENCODING)
     end
+
+    # It converts the name, data, attributes into the array of Message objects
```

Comment:
You should change this to "Converts the name, data, attributes into the array of Message objects"

Reply:
Fixed